cql3/restrictions/multi_column_restriction.hh

```diff
@@ -435,66 +435,6 @@ private:
             return std::nullopt;
         }
     }
-
-    /**
-     * Creates a single column restriction which is either slice or equality.
```

Comment:
If you like, you can fire off such commits in individual pull requests so we can retire them early.